lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -622,6 +597,85 @@ describe('GatherRunner', function() {
   });
 
   describe('artifact collection', () => {
+    it('runs gatherer lifecycle methods strictly in sequence', async () => {
```

Comment:
as we switch from promise reduce chains to for loops, want to make sure our gatherers never execute in parallel :)